learn/wavemaker-release-notes/v10-6-3.md

```diff
@@ -11,6 +11,7 @@ sidebar_label: "Release v10.6.3"
 
 - Introduced a new option, `Show At`, in the Prefab UI configuration. When using a Prefab in another application or a Prefab, all Prefab elements' UI properties display by default when you select the Prefab element. Note, when you choose a widget from the `Show At` list, the UI property will display when you select that UI property.
 - Included an option for Number and Currency widgets to limit the decimal value entry.
+- Improved the Form interaction to include a Floating button for Inline layout of Form Widget.
```

Comment:
doesn't sound right. @aparnasj your inputs here

Reply:
Introduced floating labels for form widgets to give Material Design like experience for form fields.

Reply:
added